security-analysis/security-analysis-api/src/main/java/com/powsybl/security/ViolationLocation.java

```diff
@@ -21,18 +17,5 @@ enum Type {
         BUS_BREAKER
     }
 
-    String getId();
-
     Type getType();
-
-    String getVoltageLevelId();
-
-    default Optional<String> getBusId() {
-        return Optional.empty();
-    }
-
-    default List<String> getBusBarIds() {
-        return Collections.emptyList();
-    }
-
 }
```

Comment:
Sans méthode, l'API sera dure à utiliser sans interface.

On pourrait ajouter getBusView(Network).getBusStream()
et getBusBreakerView(Network).getBusStream()

Reply:
@olperr1 @geofjamg small addition to make API more user friendly
https://github.com/powsybl/powsybl-core/pull/3178/commits/aa833ab7c1f2ea35599f613d39c31f8fd7c872c9

Can you have a look ?

Reply:
Thanks for your addition!